pkg/tsdb/azuremonitor/azuremonitor.go

```diff
@@ -128,6 +137,11 @@ func (e *AzureMonitorExecutor) DataQuery(ctx context.Context, dsInfo *models.Dat
 		return plugins.DataResponse{}, err
 	}
 
+	argResult, err := argDatasource.executeTimeSeriesQuery(ctx, azureResourceGraphQueries, *tsdbQuery.TimeRange)
```

Comment:
Not all arg queries are time series queries. Does executeTimeSeriesQuery support non-time series queries?

Reply:
executeTimeSeriesQuery interpolate Macros into KQL  and it takes time range as a parameter.